### DISCUSSION TITLE
Enhancement Request

### DISCUSSION THREAD

**Discussion by Sailfish** [audiamus/BookLibConnect#2] [Ideas]
I have a SSD as my C-drive of limited capacity. As such, to the extent possible, I choose a non-C-drive for program installations. If possible, could you consider allowing that feature?

**Comment by audiamus** [comment]
Well, always installing to the default location was by intention.

I plan to publish a new and probably the last version of AAX Audio Converter generation 1 which will check on start-up whether Book Lib Connect is already installed and if not, offer the user to download it. And for that I want to know the installation path without having to search the Windows registry for the Inno Setup installation entry. The footprint on C: for the program is rather small, just 22MB. Net 5 alone takes much more.  

With AAX Audio Converter generation 2, I will again allow to install anywhere.

**Comment by Sailfish** [comment]
I understand and thanks for the timely and detailed explanation.
